JAGPropertyConverter/JAGPropertyConverter.m

```diff
@@ -424,20 +435,39 @@ - (void) setPropertiesOf: (id) object fromDictionary: (NSDictionary*) dictionary
     for (NSString *dictKey in dictionary) {
         
         NSString *key = dictKey;
-        
-        property = [JAGPropertyFinder propertyForName: key inClass:[object class] ];
+        BOOL isKeyPath;
+        NSString *remainingKeyPath = @"";
```

Comment:
**always** initialize local variables.

remainingKeyPath should be initialized with `nil` instead of empty string.